google/resource_compute_instance.go

```diff
@@ -1160,14 +1170,11 @@ func resourceComputeInstanceUpdate(d *schema.ResourceData, meta interface{}) err
 		if val, ok := d.GetOk(prefix + ".automatic_restart"); ok {
 			scheduling.AutomaticRestart = googleapi.Bool(val.(bool))
 		}
-
-		if val, ok := d.GetOk(prefix + ".preemptible"); ok {
-			scheduling.Preemptible = val.(bool)
-		}
-
+		// The Preemptible field cannot be updated. The instance must be recreated.
```

Comment:
Hmm, so I think the following scenario would fail:
- preemptible is true
- I want to change one of the other attributes
because I think this would try to update preemptible to false.

However, we can't actually change either of the other attributes when preemptible is true- it would fail server-side. So two options for fixing this:
1. put scheduling.Preemptible back in the request- we know that we won't hit this if it changes since it's marked `ForceNew`, so this is just to propagate the current value
2. throw an error when trying to update other parts of scheduling if preemptible is true

I think #1 is clearer in the code and more future-proof, so I recommend that one- up to you though.